server/src/models/bits.rs

```diff
@@ -0,0 +1,311 @@
+/* Copyright 2022 The ModelarDB Contributors
```

Comment:
Over time it will probably be useful to decompose ModelarDB itself into multiple crates like [InfluxDB_IOx](https://github.com/influxdata/influxdb_iox) and [Apache DataFusion](https://github.com/apache/arrow-datafusion) to reduce compile time and allow reuse of components outside of ModelarDB. For example, compressed time series could be transferred to a client that embeds [`crate::models`](https://github.com/ModelarData/ModelarDB-RS/tree/master/server/src/models).

Reply:
Sounds good, I have added a [work item](https://dev.azure.com/ModelarData/ModelarDB%20Implementation/_workitems/edit/253) to do this later.